README.md

```diff
@@ -0,0 +1,43 @@
+# golang-training-enterprise
+
+## Description
+
+##### Application for enterprise
+
```

Comment:
What about preparing DB?
Add requirements for running your application
I'll get just server failed message in case run app without any db and env vars...